src/logger.ts

```diff
@@ -33,6 +33,10 @@ interface LoggerWithLogMethod extends Logger {
 export interface Logger extends BaseLogger {
     /**
      * Create a child logger.
+     * This child will apply the methodFactory of the parent. So any log extensions applied to the parent
```

Comment:
This linebreak does nothing. Was it your intention to start a new paragraph? (I think it would make sense to do so.) If so, remember you will need a blank line.